src/Components/Forms/FormCheckbox/FormCheckbox.tsx

```diff
@@ -8,19 +8,19 @@ type Formik = {
 
 const FormCheckbox: React.FC<CheckboxProps> = ({id, disabled, ...rest}) => {
   const {setFieldValue, errors, touched, values} = useFormikContext<Formik>();
-  const isError = !!errors[id!] && touched[id!] && !disabled;
+  const isError = !!errors[id!] && touched[id!] && !disabled; // NOSONAR
```

Comment:
Why are there so many NOSONAR comments?
Is it possible to address and resolve some of these issues rather than using suppressions?

Reply:
In this file I have removed most of the NOSONAR and replaced them with nullish coalesce operartor